e2e_test/run_all_tests.sh

```diff
@@ -32,6 +34,14 @@ cast send --async --json --private-key "$ACC_PRIVKEY" "$TOKEN_ADDR" 'transfer(ad
 failures=0
 tests=0
 for f in test_*"$TEST_GLOB"*; do
+	if [[ -n $NETWORK ]]; then
+		case $f in
+		  # Skip tests that require a local network.
+		  test_fee_currency_fails_on_credit.sh|test_fee_currency_fails_on_debit.sh|test_fee_currency_fails_intrinsic.sh)
```

Comment:
This is correct, but the combination of bash's case syntax and the regex made me look for the left parenthesis far too long. (no change need, just leaving random thoughts)

Reply:
It's a bit weird isn't it!